src/screens/OtherProfile/index.js

```diff
@@ -479,13 +400,13 @@ const OtherProfile = () => {
     try {
       const processPostApi = await unblockUserApi({userId: dataMain.user_id});
       if (processPostApi.code === 200) {
-        checkUserBlockHandle(dataMain.user_id);
+        refetchBlockStatus();
         blockUserRef.current.close();
         specificIssueRef.current.close();
         reportUserRef.current.close();
       }
     } catch (e) {
-      checkUserBlockHandle(dataMain.user_id);
+      refetchBlockStatus();
       blockUserRef.current.close();
       specificIssueRef.current.close();
       reportUserRef.current.close();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [392-409]

The function `checkUserBlockHandle` has been replaced with `refetchBlockStatus`. Make sure that `refetchBlockStatus` is implemented correctly and covers all the functionality provided by `checkUserBlockHandle`.



<!-- This is an auto-generated comment by OSS CodeRabbit -->